src/messages.cpp

```diff
@@ -512,7 +529,7 @@ void Messages::dialog::init( ui_adaptor &ui )
     w_fh_width = w_width;
     w_fh_x = w_x;
     help_text = filter_help_text( w_fh_width - border_width * 2 );
-    w_fh_height = help_text.size() + border_width * 2;
+    w_fh_height = help_text.size() + static_cast<unsigned long>( border_width ) * 2;
```

Comment:
1. That `unsigned long` should've been `size_t`. They are not  interchangable since we're dealing with cross-platform code, both types depend on OS and bitness and not always have same size (e.g. Win x64 has 8-byte `size_t` and 4-byte `unsigned long`)
2. Calculating in `int` is preferred in the codebase
3. The value gets implicitly cast back to `int` in this case, so we might as well deal in `int`s 
```suggestion
    w_fh_height = static_cast<int>( help_text.size() ) + border_width * 2;
```